Automattic/jetpack
# YouTube embeds: avoid popup errors caused by COOP

Fixes #33593

## Proposed changes:

YouTube embeds can be used to open popups to watch the video on YouTube, in a new window. We need to explicitely allow those popups.

More information in this Chromium bug report:
https://bugs.chromium.org/p/chromium/issues/detail?id=1463173#c13

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Go to Jetpack > Settings and enable the Shortcodes module
* Go to Posts > Add New
* On a new line, paste a YouTube video URL, e.g. `https://www.youtube.com/watch?v=o-iJEjG7QpA`
* Publish your post.
* Open that post in an incognito window.
* Click on the "Watch on YouTube" banner appearing at the bottom of the video.
* A new window should open where the video would play.
